File: Coding Practice/DSA/Python DSA/bubble_sort01.py
arr = [10, 8, 2, 6, 4]
swap = True

while swap == True:
    swap = False
    for i in range(len(arr) - 1):
        if arr[i] > arr[i + 1]:
            swap = True
            temp = arr[i]
            arr[i] = arr[i + 1]
            arr[i + 1] = temp

print(arr)

File: Coding Practice/DSA/C++ DSA/bubble_sort01.cpp
#include<iostream>

using namespace std;

int main()
{
    int arr[5] = {10, 2, 5, 3, 8};
    int temp = 0;
    bool swap = true;

    while (swap==true)
    {
        swap = false;

        for(int i = 0;i<=4;i++)
        {
            if(arr[i]>arr[i + 1])
            {
                swap = true;
                temp = arr[i];
                arr[i] = arr[i + 1];
                arr[i + 1] = temp;
            }
        }
    }
    
    for(int i=0;i<=4;i++)
    {
        cout<<arr[i]<<endl;
    }

    return 0;
}